### DISCUSSION TITLE
Any help how to create a settings page with tabs layout?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#7644] [Help]
I want help to create a settings page with tabs layout eatch tab contain a configuration of my app.

```
- Settings [
  - General
  - Logo & Favicon
  - Color Schemes
]
```

**Comment by danharrin** [comment]
Have you tried? You can use the Tabs component in the form schema.

**Reply by devhoussam1998** [reply]
I create a resource called **SettingsResource** I add Tabs in form schema but it displayed in **ListSettings** I want the tabs in **admin/settings** not in **admin/settings/create**

**Reply by devhoussam1998** [reply]
https://github.com/filamentphp/filament/assets/121255432/ff8a0f11-161f-4446-a015-e119368c97a2

**Reply by danharrin** [reply]
Make a page, not a resource, and use the form builder

**Reply by devhoussam1998** [reply]
I will test that thank you @danharrin

**Comment by devhoussam1998** [comment]
> Make a page, not a resource, and use the form builder

**Filament/Pages/Settings.php**

```
<?php

namespace App\Filament\Pages;

use Filament\Pages\Page;
use Filament\Forms\Form;

class Settings extends Page
{
    protected static ?string $navigationGroup = 'Configuration';

    protected static ?string $navigationIcon = 'bi-gear';

    protected static ?string $activeNavigationIcon = 'bi-gear-fill';

    protected static ?string $navigationLabel = 'Settings';

    protected static ?int $navigationSort = 12;

    protected ?string $subheading = 'Custom Page Subheading';

    protected static string $view = 'filament.pages.settings';

    public function form(Form $form): Form
    {
        return $form
            ->schema([
                Forms\Components\TextInput::make('title'),
            ]);
    }
}
```
Here is what i get as a result:

<img width="1292" alt="Screenshot 2023-08-10 at 18 06 40" src="https://github.com/filamentphp/filament/assets/121255432/f28f45db-51c6-4374-9d32-74303df05ec5">

as you can see **TextInput** not displayed?

**Reply by danharrin** [reply]
so how are you rendering the form in Blade? Please follow *all* the instructions in `Adding a form to a Livewire component` doc
